src/picknik_ur_base_config/config/base_config.yaml

```diff
@@ -38,29 +38,23 @@ hardware:
   ip: "!!! Please set the IP address of the robot"
 
   # Specify additional launch files for running the robot with real hardware.
-  # [Required]
+  # [Optional, defaults to a blank launch file if not specified]
```

Comment:
Are we planning on using these tags for anything automated? I only ask because if so the second part would be better served as a comment. If these are just comment tags anyways I think this is fine.

Reply:
I don't think they're used for anything automated, they were there just to pop out for users reading the config files... but I see your point for consistency.